lib/controllers.js

```diff
@@ -21,7 +21,7 @@ module.exports = function (rootDirectory, logger, indexDocument, errorDocument,
     res.header('Access-Control-Allow-Origin', '*');
     res.header('Etag', '"' + object.md5 + '"');
     res.header('Last-Modified', new Date(object.modifiedDate).toUTCString());
-    res.header('Content-Type', object.contentType);
+    res.setHeader('Content-Type', object.contentType);
```

Comment:
See: https://github.com/expressjs/express/issues/2697

AWS does not append the charset to the content-type.